apps/sparo-lib/src/services/GitSparseCheckoutService.ts

```diff
@@ -266,6 +271,7 @@ export class GitSparseCheckoutService {
   private _prepareMonorepoSkeleton(options: { restore?: boolean } = {}): void {
     const { restore } = options;
     this._finalSkeletonPaths = this._getSkeletonPaths();
+    this._terminalService.terminal.writeLine('Checking out skeleton...');
```

Comment:
Seems this log will be shown in terminal every time when we run any sparo command.

As every time we invoke `ensureSkeletonExistAndUpdated`, the `_isSkeletonInitializedAndUpdated` is always false at the first time, and will run `_prepareMonorepoSkeleton`

Reply:
If we remove this line. The log will be 

```
Checking out and updating core files...

--[ git sparse-checkout ]----------
-----------------------------------

--[ git sparse-checkout ]----------
-----------------------------------
```

Adding this line seems to make more sense at this moment. The log will be 

```
Checking out and updating core files...

--[ git sparse-checkout ]----------
-----------------------------------

Checking out skeleton...

--[ git sparse-checkout ]----------
-----------------------------------
```

------------

Yes. You are right. The log is also printed every time running any command. That should be another problem:

Q: Is it necessary to run "git sparse-checkout" twice every time?
Q: If it's necessary. Do we want to hide the underlaying information to our users?